en/components/combo_features.md

```diff
@@ -87,27 +87,134 @@ export class ComboDemo implements OnInit {
+> [!Note]
+> If [displayKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#displaykey) is omitted then [valueKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#valuekey) entity will be used instead.
 
 
 Follow this [topic](combo_remote.md) for more details about binding [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) with remote data.
 
+
+## Selection
+The combo handles selection differently, depending on its [valueKey]({environment:angularApiUrl}/classes/igxcombocomponent.html#valuekey) property.
+
+### Selection with valueKey
+If a `[valueKey]` is provided, all of the combo's selection events will be handled with the value of the items' `valueKey` property.
+For example, if we have the following combo defined:
+ ```html
+ <igx-combo [data]="towns" valueKey="id"></igx-combo>
+ ``` 
+ ```typescript
+ export class MyBoundCombo {
+	@ViewChild(IgxComboComponent, { static: false, read: IgxComboComponent })
+	public combo: IgxComboComponent; 
+	public towns: { name: string, id: string } = [{ name: "Sofia", id: "BG01" }, { name: "London", id: "UK01" }, ...];
+ }
+ ```
+
+Selection is handled using **only the `id` property** (specified in the `valueKey` input) of the items:
+```typescript
+    // set
+    this.combo.selectItems(["UK01", "UK02", "UK03"]);
+    // get
+    const selectedItems: string[] = this.combo.selectedItems();
+```
+As you can see, the call to [`selectItems`]({environment:angularApiUrl}/classes/igxcombocomponent.html#selectitems) is done with an array of `string`s (since that's expected to be in the `id` field of entries in `towns: Towns[]`) and `selectedItems` returns a `string[]`, containing the `id`s of the selected towns.
+
+The above is true for **all** of the combo events and methods that are related to the selection:
+ - [`selectItems`]({environment:angularApiUrl}/classes/igxcombocomponent.html#selectitems), [`deselectItems`]({environment:angularApiUrl}/classes/igxcombocomponent.html#deselectitems), [`selectAllItems`]({environment:angularApiUrl}/classes/igxcombocomponent.html#selectallitems), [`deselectAllItems`]({environment:angularApiUrl}/classes/igxcombocomponent.html#deselectallitems) - these methods should be called with the corresponding `valueKey` property values if `[valueKey]` is specified for the combo.
+ - [`onSelectionChange`]({environment:angularApiUrl}/classes/igxcombocomponent.html#onselectionchange) - the `newSelection`, `oldSelection`, `added` and `removed` arrays of the [`IComboSelectionChangeEventArgs`]({environment:angularApiUrl}/interfaces/icomboselectionchangeeventargs.html) will hold the corresponding values of the items. 
+
+### Selection without valueKey
+When no `[valueKey]` is provided, the combo handles selection based on **equality (===)**.
+If the data is of a primitive type (e.g. `string`, `number`), the provided array to selection methods should contain only the values that you'd like to select. For example:
```

Comment:
I think "you would" be more suitable. If you decide so - change this everywhere in the topic.